### ISSUE TITLE
Is there a way to toggle visibility on subLayers?

### ISSUE THREAD

**Issue by bishoym** [uber/nebula.gl#541]
The [DeckGL docs](https://deck.gl/docs/developer-guide/performance/) recommend using the [visibility](https://deck.gl/docs/developer-guide/performance#favor-layer-visibility-over-addition-and-removal) prop where frequent toggling of data is necessary. 

If I'm using the `EditableGeoJSONLayer`, and passing it a `FeatureCollection`, is there a way to access this `visible` prop on a **specific feature within the features array** (by index)?

The visible prop at the top level toggles the entire Layer - and therefore _all the features_ simultaneously. That's not quite the desired behaviour. Any help is much appreciated!

**Comment by supersonicclay** [comment]
You can provide deck.gl layers callbacks for changing how things are rendered. If you pass `_subLayerProps` to `EditableGeoJsonLayer`. And in those props, you can pass something like this:
```
const [myFeatures, setMyFeatures] = React.useState(...);

const layer = EditableGeoJsonLayer({
_subLayerProps: {
  geojson: {
    getLineColor: (feature) => {
      if (myFeatures.features.indexOf(feature) === INDEX_YOU_WANT_TO_HIDE) {
        return TRANSPARENT_COLOR;
      } else {
       return NORMAL_COLOR;
     }
   }
 });
 ```

**Comment by bishoym** [comment]
@supersonicclay yup, I've gotten quite familiar with the API surface of this particular class over the last couple of months 😅 and I've implemented your suggestion. 

Good news is: It technically works. Bad news is: it's hacky and requires a refactor of vertex styling in order to avoid having a set of floating vertices in EditMode. 
![image](https://user-images.githubusercontent.com/22736690/111910931-0af71c80-8a6c-11eb-9e17-3de740122ce6.png)


Also, more crucially, it doesn't stop me from **picking** the layer which is unfortunately where I have to determine the solution as insufficient.

Like I said, I need access to the `visible` prop and that _unfortunately_ doesn't support a callback. 

I know we can extend the `EditableGeoJSONLayer` class, can you provide guidance as to how I might go about overriding the top-level `visible` prop from boolean to a callback which iterates over the underlying features?

**Comment by MoxiCult** [comment]
> 

bishoym - did you find a solution? I am having a similar issue.
